### DISCUSSION TITLE
How to make my launcher open search engine page in maximized window?

### DISCUSSION THREAD

**Discussion by Unintendedz** [Flow-Launcher/Flow.Launcher#1926] [Q&A] [answered]
Hi everyone, I have a question about my launcher. I usually use it to quickly search for something on the internet, and it works great. However, I'd like to know if there's a way to make the launcher open my search engine in a maximized window by default.

Currently, the browser window that opens is not maximized, which is a bit inconvenient. I have to manually maximize it every time I use the launcher, and it would be great if I could skip this step.

**Comment by jjw24** [comment, marked as answer]
Hmm, I maybe wrong but this sounds like a windows thing that is specific to your windows configuration? Flow doesn't do anything with browser window size, it just opens the default browser.

**Reply by Unintendedz** [reply]
Thanks for the reply! You were totally right, it is a windows thing. I manually resized the browser instead just maximized it, and it will keep the size now.  Appreciate for giving me the idea!
